osquery/tables/networking/windows/process_open_sockets.cpp

```diff
@@ -5,12 +5,36 @@
  *  This source code is licensed in accordance with the terms specified in
  *  the LICENSE file found in the root directory of this source tree.
  */
+#include <string>
+#include <vector>
 
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 
 #include "win_sockets.h"
 
+namespace {
```

Comment:
I think we usually put the anon namespaces within the `osquery::tables` namespace? I am just looking at other osquery code and trying to infer what is considered 'consistent'. Could we move this to line 40?

Reply:
Done